.github/workflows/nightly.yml

```diff
@@ -0,0 +1,62 @@
+name: nightly
```

Comment:
So these are _just_ the daemon tests, correct? I'm not sure they warrant a separate pipeline, or if they should be a step in the "regular" CI.

Since this is a much smaller project than `aiida-core`, I'd like to keep the testing infrastructure at a smaller level also.

Reply:
Fair point, I move it to ci.yml.